src/viur/core/skeleton.py

```diff
@@ -1202,7 +1241,7 @@ def fixDotNames(entity):
 
         # END of __txn_update subfunction
 
-        # Run our SaveTxn
+        # Run transactional function
```

Comment:
IMO comments like this are unnecessary if you write self-explaining code and can be removed completely.
A more helpful comment would be: _"Run the function in a transaction or inline if we're already in a transaction"_. But even that is self-explaining and uncessary, imo.




This is the same like:
```py
# Set variable foo to 3
foo = 3
```

Reply:
the comment was there before and related to a function which doesn't exist since 12 years or so. Your comment on the comment is just as useless.

Reply:
Yes, so just remove it instead of `verschlimmbessern` it.
```suggestion
```